tests/e2e/e2e_upgrade_test.go

```diff
@@ -49,7 +49,7 @@ var _ = OSMDescribe("Upgrade from latest",
 			// release branches, it should specify the most recent patch of the
 			// previous minor release. e.g. on the release-v1.0 branch, this
 			// should be "0.11".
-			i.Version = ">0.0.0-0"
+			i.Version = ">0.1.0-0"
```

Comment:
Like the previous release branch: https://github.com/openservicemesh/osm/blob/26886c4f16ae5cc92ddfe4c62c87cd82f1cd965c/tests/e2e/e2e_upgrade_test.go#L51
```suggestion
			i.Version = "1.0"
```